pub_sub/go/sdk/order-processor/app.go

```diff
@@ -18,7 +19,13 @@ var sub = &common.Subscription{
 }
 
 func main() {
-	s := daprd.NewService(":6001")
+	//read app-port passed thru command line
+	appPort, isSet := os.LookupEnv("APP_PORT")
```

Comment:
```suggestion
	if appPort, ok := os.LookupEnv("APP_PORT"); !ok {
	        // fail
	}
```

Reply:
@shubham1172 the intent is to initialize and declare the variables based on os.Lookup. Including them in 'if' would require a declaration of appPort since it is used outside the if condition. I am not in favour of adding an else condition since it impacts readability of the simple flow. Will leave this as-is.